i18n/en.json

```diff
@@ -361,7 +361,7 @@
   },
   {
     "id": "api.channel.remove_member.removed",
-    "translation": "%v was removed from the team."
+    "translation": "%v was removed from the channel."
```

Comment:
Hi @hmhealey. This resolved the case 1 of #6483. Let me know if I should create a new message or leave it as is. I was not really sure about it so I added it anyway.